CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [1.2.0] - 2022-06-03
+
+### Changed 
+- Rename the package name from ndart to ndart_cli in order to free up this package name 
+  for the main angular package
+- small dependency packages version updates
+- set // @dart=2.9 in main.dart to prevent null sound issue messages until ngcomponent will be updated
```

Comment:
Remove this line, capitalize the first letter of every sentence, and then we're good to go :D

Reply:
ok, updated